code/__DEFINES/components.dm

```diff
@@ -142,7 +142,7 @@
 #define COMSIG_TRY_STORAGE_FILL_TYPE "storage_fill_type"			//(type, amount = INFINITY, force = FALSE)			//don't fuck this up. Force will ignore max_items, and amount is normally clamped to max_items.
 #define COMSIG_TRY_STORAGE_TAKE "storage_take_obj"					//(obj, new_loc, force = FALSE) - returns bool
 #define COMSIG_TRY_STORAGE_QUICK_EMPTY "storage_quick_empty"		//(loc) - returns bool - if loc is null it will dump at parent location.
-#define COMSIG_TRY_STORAGE_RETURN_INVENTORY "storage_return_inventory"	//(list/list_to_inject_results_into)
+#define COMSIG_TRY_STORAGE_RETURN_INVENTORY "storage_return_inventory"	//(list/list_to_inject_results_into, recursively_search_inside_storages = TRUE)

```

Comment:
Why this if you don't use it.

Reply:
It'll be used in the future though...

Reply:
okay now that i'm not on a phone and asleep the reason i left it in is because storage was supposed to be able to be used by things that aren't players by using component signals, and being able to see its inventory (which isn't always the same as it's contents list) is an important interface i guess.

Reply:
Don't mix fixes with interface expansion please.